packages/bueno/project.json

```diff
@@ -1 +1,109 @@
-{}
+{
```

Comment:
I don't think any optimization is necessary for bueno, its lightweight and quick. Perhaps we'll replace it with zod as well

Reply:
Good point, I should probably introduce some "good enough" defaults for smaller packages like this.